article/_posts/2019-07-29-Dont-Worry-About-The-Weather.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: review
+title: Don't Worry About the Weather Unsupervised Condition-Dependent Domain Adaptation
+tags: deep-learning CNN segmentation domain-adaptation GAN unsupervised traffic
+cite:
+    authors: "Horia Porav, Tom Bruls, Paul Newman"
+    title:   "Don't Worry About the Weather: Unsupervised Condition-Dependent Domain Adaptation"
+    venue:   "Arxiv, submitted on 25 Jul 2019"
+pdf: "https://arxiv.org/pdf/1907.11004"
```

Comment:
Although I ignore how strict we are concerning this, it "should be" a link to the PDF if we want to honor the label.

Reply:
Done in 07b72cb.